packages/vue/src/examples/pages/checkout/Checkout.vue

```diff
@@ -3,36 +3,38 @@
     <div class="checkout">
       <div class="checkout__main">
         <SfSteps :active="currentStep" @change="updateStep($event)">
-          <SfStep name="Personal Details">
+          <SfStep name="Details">
             <PersonalDetails
-              :order="order"
-              @update:order="updateOrder($event)"
+              v-model="personalDetails"
```

Comment:
@martaradziszewska do you think a v-model is the right way? IMO we should use props and $event to be more transparent, WDYT?